components/brave_rewards/browser/rewards_service.h

```diff
@@ -183,6 +190,34 @@ class RewardsService : public KeyedService {
   virtual void GetConfirmationsHistory(
       brave_rewards::ConfirmationsHistoryCallback callback) = 0;
 
+  virtual void GetAutoContributeCount(
+      const rewards_counter::OnRewardsCountedCallback callback) = 0;
+
+  virtual void ClearAutoContributeData(
+      OnRemoveRewardsDataCallback callback,
+      uint64_t reconcile_stamp) = 0;
+
+  virtual void OnClearAutoContributeData(
```

Comment:
this also doesn't need to be here - see OnGetIsContributionInProgress

Reply:
Removed